stripe/resource_stripe_plan.go

```diff
@@ -24,6 +24,7 @@ func resourceStripePlan() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				Computed: true,
+				ForceNew: true,
```

Comment:
This is another thing I ran into, I wanted to set a "readable name" for the plan in the end, and terraform failed because it tried to `POST /plans/hash-id` with `plan_id` in the body. So basically, it isn't possible per API docs to edit this, it should force a new Stripe plan instead.